app/views/trainees/training_routes/_form_fields.html.erb

```diff
@@ -7,15 +7,13 @@
   <% else %>
     <%= f.govuk_radio_button :training_route, TRAINING_ROUTE_ENUMS[:assessment_only], label: { text: t("activerecord.attributes.trainee.training_routes.assessment_only") }, link_errors: true %>
 
-    <% TRAINING_ROUTE_FEATURE_FLAGS.each do |training_route| %>
-      <% if FeatureService.enabled?("routes.#{training_route}") %>
-        <%= f.govuk_radio_button(
-          :training_route,
-          TRAINING_ROUTE_ENUMS[training_route],
-          hint: { text: (TRAINING_ROUTE_ENUMS[training_route].starts_with?("provider_led") && t(".provider_led_hint")) },
```

Comment:
Moved into helper to take this logic out of the view now we have multiple hints for multiple routes.